### ISSUE TITLE
Clearing the navigation history, without using the "app:///NavigationPage/"

### ISSUE THREAD

**Issue by KSemenenko** [PrismLibrary/Prism#1750]
I have a situation when I need to clear the navigation history.
I found a solution to open a new page. and thereby erase history.
But this option does not suit me, since during the creation of the page I see a white screen for a while.
Therefore, I would like to be able to simply clear the navigation stack.
Is there such an options?

**Comment by dansiegel** [comment]
Closing as Spam

**Comment by KSemenenko** [comment]
then let's continue only here @dansiegel 
How can I clear the navigation stack?

**Comment by dansiegel** [comment]
Questions belong on Stack Overflow

**Comment by KSemenenko** [comment]
Yes, I have seen, it is advised to do this here with "app: /// NavigationPage /" which is equivalent to 
``` cs 
MainPage = new NavigationPage (new SomePage);
```

but there is a problem with this method, a white screen while the page is being created.
So I found a way to do this through the NavigateTo navigation, and then clear the navigation stack.
but I did not find how to do it in Prism.

Application.Current.MainPage also does not give full access to the navigation stack.

Maybe I’m missing something, and you obviously have a way to do this in this framework.
